s3/src/main/resources/reference.conf

```diff
@@ -77,8 +77,13 @@ alpakka.s3 {
   # AWS S3 is going to retire path-style access, thus prefer the virutal-host-style access
   # https://aws.amazon.com/blogs/aws/amazon-s3-path-deprecation-plan-the-rest-of-the-story/
   path-style-access = false
+  # Don't use! Possibility to switch off the logged warning when using `path-style-access`.
+  path-style-access-warning = true
 
   # Custom endpoint url, used for alternate s3 implementations
+  # To enable DNS-style access with Alpakka S3 use the placeholder `{bucket}` in the URL
```

Comment:
Maybe use "virtual-host-style" access or whatever the S3 documentation uses consistently here.